cpp/modmesh/universe/World.cpp

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
This file `World.hpp` is currently empty but I will add code in the future.  If I lie, I will delete the file.